omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -24,30 +24,15 @@ fi
 if [[ $type == 'CS' ]];
 then
     path='opscode'
-    ctl_cmd='private-chef-ctl'
+    ctl_cmd='chef-server-ctl'
     config_name='*chef*.rb'
     ha=$(egrep -qs 'topology[[:space:]]+.*ha' /etc/opscode/*chef*.rb)$?
 
     if [[ ! -e "/opt/$path/bin/$ctl_cmd" ]];
     then
-        echo "ERROR: Enterprise or Chef Server may not be installed."
+        echo "ERROR: Chef Server may not be installed."
         exit 1
     fi
-
-    echo "Gathering logs from Enterprise or Chef Server."
-elif [[ $type == 'OSC' ]];
```

Comment:
Are we sure the support team doesn't still need this?  I believe they might instruct users to download this file from the repository if they happen to be running a version that of Chef 11 that didn't ship with this.